### PR TITLE
Change MultiWayIf layout to keep consistent 2-space indent increase

### LINKED ISSUE #300: What is the policy for odd (2k+1) indentation?
MultiWayIf is formatted such way:
```haskell
if | condition -> _
   | condition -> _ -- 3 spaces
   | condition ->
     continuation -- 5 spaces
       and_so_on -- 7 spaces
```

Is it intended?

### PR DESCRIPTION
**Pull request by cblp**
Close #300

### CODE REVIEW

File: src/Ormolu/Printer/Meat/Declaration/Value.hs [hunk 1]
```diff
@@ -549,8 +549,9 @@ p_hsExpr = \case
       breakpoint
       inci (p_hsExpr x)
   HsMultiIf NoExt guards -> do
-    txt "if "
-    sitcc $ sep newline (located' (p_grhs RightArrow)) guards
+    txt "if"
```
Comment (mrkkrp) [inline src/Ormolu/Printer/Meat/Declaration/Value.hs]: Here, notice we first output the literal `"if"`, then we need either single space or newline, this is called `breakpoint`. Then we need to make sure that the rest is more indented, so we use `inci`. It has no effect in single-line case, and bump indentation as you'd expect in multiline case.

Comment (cblp) [reply src/Ormolu/Printer/Meat/Declaration/Value.hs]: Thanks! I will try to use them in future patches.

**Comment by cblp** [comment]
@mrkkrp, you rewrote whole my change, and now I'm not its author.

**Comment by mrkkrp** [comment]
You're the author and I'm the committer, technically.

**Pull request merged**